migrations/kubernetes-agent/KubernetesAgent.IntegrationTests/Utils/UpgradeManagerWrapper.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Text;
+using Halibut;
+using KubernetesAgent.Integration.Setup.Common;
+using KubernetesAgent.UpgradeManager;
+using Octopus.Tentacle.Client;
+using Octopus.Tentacle.Client.Scripts.Models;
+using Octopus.Tentacle.Client.Scripts.Models.Builders;
+using Octopus.Tentacle.Contracts;
+using Octopus.Versioning;
+using File = KubernetesAgent.UpgradeManager.File;
+
+namespace KubernetesAgent.Integration.Utils
+{
+    public class UpgradeManagerWrapper
```

Comment:
This guy just provides a way to execute the UpgradeManagers scripts and give back the output of that script (and not all the other messages/service messages that the agent spits out when executing a script.)